src/components/ContactsList/ContactsList.jsx

```diff
@@ -60,7 +60,8 @@ ContactsList.propTypes = {
   contacts: PropTypes.array,
   onClickContact: PropTypes.func,
   onSelect: PropTypes.func,
-  selection: PropTypes.array
+  selection: PropTypes.array,
+  displayImportation: PropTypes.func
```

Comment:
This `displayImportation` function goes down very deep.
I don't know why this is not under the responsibility of the empty list component, or more accurately, a child of the empty list component. But I guess it could be.

Reply:
Because as soon as you start importing contacts, the root component doesn't display the empty list anymore.

Reply:
So the root component wants to know about the contacts list content, not something about the import modale.

Reply:
In other words: the import modal can decide on its very own if it needs to be displayed or not, given the contacts list length.

Reply:
```js
render () {
  <div class="main">
    <ContactsList />
    { isEmpty(contacts)
      ?  <ImportationModale />
      :  null
    }
}
```

Reply:
I remember why I didn't handle the importation modal visibility in `ContactsList`: because `ContactsEmptyList` and `ContactsList` are rendered separately. I'll handle visibility in `ContactApp` for now, it shouldn't trigger useless rendering anyway since the importation state handling will be moved to the modal.